manifests/image-references

```diff
@@ -102,3 +102,11 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-kube-rbac-proxy:latest
+  - name: csi-driver-shared-resource-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-csi-driver-shared-resource-operator:latest
+  - name: csi-driver-shared-resource
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-csi-driver-shared-resource:latest
```

Comment:
/hold

We can't add these image references until we have the ART pipeline set up and have an image built in OSBS. Otherwise we break the payload and block nightly releases.